resources/tag_models/event.json

```diff
@@ -0,0 +1,29 @@
+{
+  "tag": {
+	"api_key": "", // String - API key - Make sure the request is from a valid client (idea nicked from Keen.io) useful if a page gets copied onto a Russian website and creates noise
+	"version": "", // Version of the tracking client e.g. "JS v1.2"
+	"type": "event", // Type of request (page or event)
+	"id": "", // ID of this request, must be unique
+	"offset": 0 // Delay of this tag between event happening and being sent to server - milliseconds
+  },
+
+  "event": {
+	"page_id": "", // ID of the page on which this event happened.
+	"parent_id": "", // Parent event ID if this is a child event.
```

Comment:
They are. We need the consequence of events, not just counts.

It is a business requirement. 

Regardless, I don't see why not have it. It makes no difference to the pipeline if there's an extra ID in there, and it provides far richer information. And allows the analytics team to do their job. 



> On 28 May 2015, at 22:17, Matt Chadburn <notifications@github.com> wrote:
> 
> In resources/tag_models/event.json:
> 
> > @@ -0,0 +1,29 @@
> > +{
> > +  "tag": {
> > +	"api_key": "", // String - API key - Make sure the request is from a valid client (idea nicked from Keen.io) useful if a page gets copied onto a Russian website and creates noise
> > +	"version": "", // Version of the tracking client e.g. "JS v1.2"
> > +	"type": "event", // Type of request (page or event)
> > +	"id": "", // ID of this request, must be unique
> > +	"offset": 0 // Delay of this tag between event happening and being sent to server - milliseconds
> > +  },
> > +
> > +  "event": {
> > +	"page_id": "", // ID of the page on which this event happened.
> > +	"parent_id": "", // Parent event ID if this is a child event.
> @markstephens IIRC the use cases for needing this sort of thing weren't that strong?
> 
> —
> Reply to this email directly or view it on GitHub.
> 

-- 

------------------------------
This email was sent by a company owned by Pearson plc, registered office at 
80 Strand, London WC2R 0RL.  Registered in England and Wales with company 
number 53723.